recipes/conversation_view/feed_pill/feed_item_pill.stories.js

```diff
@@ -61,66 +65,38 @@ const DefaultTemplate = (args, { argTypes }) => createTemplateFromVueFile(
 // Stories
 export const Default = {
   render: DefaultTemplate,
-  parameters: { a11y: { disable: true } },
-  args: {
-    iconName: 'video',
-    title: 'This meeting has ended',
-    ariaLabel: 'Click to expand',
-    buttonClass: '',
-    toggleable: true,
-    wrapperClass: 'd-w628',
-    borderColor: 'default',
-  },
+  parameters: { options: { showPanel: false }, a11y: { disable: true }, controls: { disable: true } },
```

Comment:
You should not disable the controls panel on default story.

```suggestion
  parameters: { a11y: { disable: true } },
```